src/bin/config/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod default_ctants;
```

Comment:
And therefore we get rid of one module as well.

Reply:
Done